Dockerfile.assisted-service

```diff
@@ -1,10 +1,36 @@
-FROM registry.access.redhat.com/ubi8/ubi-minimal:latest as certs
+# Generate python client
+FROM swaggerapi/swagger-codegen-cli:2.4.15 as swagger_py
 
+COPY swagger.yaml .
+COPY tools/generate_python_client.sh .
+RUN chmod +x ./generate_python_client.sh && SWAGGER_FILE=swagger.yaml OUTPUT=/build ./generate_python_client.sh
+
+# Generate go client
+FROM quay.io/goswagger/swagger:v0.25.0 as swagger_go
+COPY . .
+RUN mkdir /build
+RUN swagger generate server --template=stratoscale -f swagger.yaml --template-dir=/templates/contrib \
+    && mv restapi models /build
+RUN swagger generate client --template=stratoscale -f swagger.yaml --template-dir=/templates/contrib \
+    && mv client /build
+
+# Build binaries
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.14 as builder
+COPY . .
+COPY --from=swagger_py /build build
+COPY --from=swagger_go /build/client client
+COPY --from=swagger_go /build/models models
+COPY --from=swagger_go /build/restapi restapi
+RUN GOFLAGS="" GO111MODULE=on go build -o /build/assisted-service cmd/main.go
+RUN cd build && python3 setup.py sdist --dist-dir /assisted-service-client
+
+# Create final image
+FROM registry.access.redhat.com/ubi8/ubi:latest as certs
 FROM scratch
 COPY --from=certs /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem /etc/ssl/certs/ca-bundle.crt
 COPY --from=certs /etc/pki/ca-trust/extracted/openssl/ca-bundle.trust.crt /etc/ssl/certs/ca-bundle.trust.crt
 ARG GIT_REVISION
 LABEL "git_revision"=${GIT_REVISION}
-ADD build/assisted-service /assisted-service
-ADD build/assisted-service-client-1.0.0.tar.gz /clients/assisted-service-client-1.0.0.tar.gz
-CMD ["/assisted-service"]
+COPY --from=builder /build/assisted-service /assisted-service
+COPY --from=builder /assisted-service-client/assisted-service-client-1.0.0.tar.gz /clients/assisted-service-client-1.0.0.tar.gz
```

Comment:
i think that the version should be a parameter or the hush or anything that is not fixed on `1.0.0`

Reply:
I wanted to use `git describe` to get the current tag, but the current repository don't have tags at all